### PR TITLE
Add "Update apps in the OpenStore"

### PR DESCRIPTION
**Pull request by UniversalSuperBox**

### CODE REVIEW

File: plugins/system-update/PageComponent.qml [hunk 1]
```diff
@@ -172,6 +172,13 @@ ItemPage {
                 }
             }
 
+            ListItem.SingleValue {
+                objectName: "updateApps"
```
Comment (mardy) [inline plugins/system-update/PageComponent.qml]: Not needed

File: plugins/system-update/PageComponent.qml [hunk 2]
```diff
@@ -172,6 +172,13 @@ ItemPage {
...
+                text: i18n.tr("Update apps in the OpenStore")
```
Comment (mardy) [inline plugins/system-update/PageComponent.qml:176]: Looks OK to me, but I wonder if "Check for app updates in the OpenStore" is better? (I'll leave it to you, as I'm not a native speaker :-) )

**Review by mardy — CHANGES_REQUESTED** [review]
I've been exploring two different possibilities:
1) Fully restoring the previous functionality
2) Follow a similar approach to this one, but show the item only if there are updates

Both approaches require a bit more time than I expected (and I'm not sure I'll be able to dedicate time to this tomorrow), so I suggest that we merge this branch.

I'll still continue working on this, and hopefully I'll have the second option implemented soon, but this is a safe fallback which we can improve later.

File: plugins/system-update/PageComponent.qml (hunk 1 above)
Comment (UniversalSuperBox) [reply plugins/system-update/PageComponent.qml]: Ah, I see it's only for testing. Interesting.

**Review by mardy — APPROVED** [review]
Excellent, thanks! Just please squash the two commits into one, before merging :-)

**Pull request merged**